### DISCUSSION TITLE
Validate notices created with sdk 1.12

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#982] [Q&A] [answered]
Hello,

Is it possible to validate notices created with sdk 1.12.0?

We are trying to validate a notice created with SDK 1.12.0 using the validate operation (API) of Swagger and we are receiving the following response:

`{
  "errorCode": 400,
  "message": "The eForms SDK version 'eforms-sdk-1.12' does not exist.",
  "timestamp": "2024-07-23T10:37:03Z"
}`

![image](https://github.com/user-attachments/assets/33487ffd-2af9-452a-a5b9-684af3750c1f)

Thanks in advance,
Alex

**Comment by YvesJo** [comment, marked as answer]
Hi,
For such non-SDK related questions, please get in touch with the helpdesk.
Please note that he new SDK 1.12 will get available in Preview starting tomorrow only.
KR

**Reply by AlexAS81** [reply]
OK thanks. We will try to validate it tomorrow. At what time will version 1.12 of the sk be available?

**Reply by bertrand-lorentz** [reply]
To complement the answer:
You can always use the API operation "version-range" provided by our validation service and available in the same place as the validation operation from the screenshot above.
This will give you the range of SDK versions currently accepted, with the date when it will or did change. For more details, see the documentation at https://docs.ted.europa.eu/eforms-common/active-versions/index.html#version-range.
The information returned is used by the validation service itself to determine if an SDK version is accepted or not, so it will always be accurate.
